misc.Rmd

```diff
@@ -2,56 +2,71 @@
 
 -   `po/`: translations for messages.
-    This is useful but beyond the scope of this book.
-    See the [Internationalization](https://cran.rstudio.com/doc/manuals/r-devel/R-exts.html#Internationalization) chapter of "R extensions" for more details.
-
--   `tools/`: auxiliary files needed during configuration, or for sources that need to generate scripts.
+    This is useful, but beyond the scope of this book.
+    See the [Internationalization](https://cran.rstudio.com/doc/manuals/r-devel/R-exts.html#Internationalization) chapter of "Writing R extensions" for more details.
 
 ## Installed files {#inst}
 
-When a package is installed, everything in `inst/` is copied into the top-level package directory.
-In some sense `inst/` is the opposite of `.Rbuildignore` - where `.Rbuildignore` lets you remove arbitrary files and directories from the top level, `inst/` lets you add them.
-You are free to put anything you like in `inst/` with one caution: because `inst/` is copied into the top-level directory, you should never use a subdirectory with the same name as an existing directory.
-This means that you should avoid `inst/build`, `inst/data`, `inst/demo`, `inst/exec`, `inst/help`, `inst/html`, `inst/inst`, `inst/libs`, `inst/Meta`, `inst/man`, `inst/po`, `inst/R`, `inst/src`, `inst/tests`, `inst/tools` and `inst/vignettes`.
+When a package is installed, everything in `inst/` is copied into the top-level directory of the installed package.
+In some sense `inst/` is the opposite of `.Rbuildignore` - where `.Rbuildignore` lets you remove arbitrary files and directories from the built package, `inst/` lets you add them.
 
-This chapter discusses the most common files found in `inst/`:
+You are free to put anything you like in `inst/` with one caution: because `inst/` is copied into the top-level directory, don't create a subdirectory that collides with any of the directories that make up the official structure of an R package.
+We recommend avoiding directories with special significance in either the source or installed form of a package, such as: `inst/data`, `inst/help`, `inst/html`, `inst/libs`, `inst/man`, `inst/Meta`, `inst/R`, `inst/src`, `inst/tests`, `inst/tools`, and `inst/vignettes`.
+In most cases, this prevents you from having a malformed package.
+And even though some of the above directories are technically allowed, they can be an unnecessary source of confusion.
 
--   `inst/AUTHOR` and `inst/COPYRIGHT`.
-    If the copyright and authorship of a package is particularly complex, you can use plain text files, `inst/COPYRIGHTS` and `inst/AUTHORS`, to provide more information.
+Here are some of the most common files and folders found in `inst/`:
 
--   `inst/CITATION`: how to cite the package, see below for details.
+-   `inst/AUTHORS` and `inst/COPYRIGHTS`: If the authorship or copyright of a package is particularly complex, you can use these plain text files to provide more information.
```

Comment:
I'm not sure that `inst/AUTHORS` is correct and I'm not sure we'd recommend `inst/COPYRIGHTS` any more. But I think you can probably just drop this bullet since we now have the licensing chapter.

Reply:
ok removed